File: JavaScript/Medium/strings/seqeuntialDigits.js
var solution = function(low, high){
    let allDigits = '123456789'
    let res = []

    for (let windowSize = low.toString().length; windowSize < high.toString().length + 1; windowSize++){

        for (let start = 0; start<10-windowSize; start++){
            end = start+windowSize
            let curDigit = parseInt(allDigits.slice(start, end))
            if (curDigit >= low && curDigit <= high) res.push(curDigit)
        }
    }

    return res
}

File: Python/Medium/strings/seqeuntialDigits.py
def solution(low, high):
    allDigits = '123456789'
    res = []
    for windowSize in range(len(str(low)), len(str(high))+1):

        for start in range(0, 10 - windowSize):
            end = start+windowSize
            curDigit = int(allDigits[start:end])
            if curDigit >= low and curDigit <= high:
                res.append(curDigit)

    return res
